media-gfx/converseen/converseen-0.9.10.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake xdg-utils
+
+DESCRIPTION="Batch image converter and resizer based on ImageMagick"
+HOMEPAGE="https://converseen.fasterland.net/
+	https://github.com/Faster3ck/Converseen/"
+SRC_URI="https://github.com/Faster3ck/Converseen/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+# FIXME: graphicsmagick dependency does not work properly, failures when compiling
```

Comment:
Any upstream issue link we could track here? Notes like these tend to carry on through many versions without any actions ever taken. If there's an upstream issue link, we can check it each time while bumping the package.